Maria/scheduler docs (#680)
telerik/maui-docs: controls/scheduler/non-working-hours.md

---

> title: Non-working Hours

Reviewer: ```suggestion
title: Nonworking Hours
```

> page_title: .NET MAUI Scheduler Documentation - Non-working Hours

Reviewer: ```suggestion
page_title: .NET MAUI Scheduler Documentation - Nonworking Hours
```

> description: Learn more about the non-working hours feature in the Telerik UI for .NET MAUI Scheduler control.

Reviewer: ```suggestion
description: Learn more about the nonworking hours feature in the Telerik UI for .NET MAUI Scheduler control.
```

published: False 
slug: scheduler-non-working-hours
---

> # Non - Working Hours

Reviewer: ```suggestion
# Nonworking Hours
```

> The Telerik UI for .NET MAUI Scheduler control provides the option to define the day working hours of the work week in  the different Views. This means that you can set work start/end times, and in this case the remaining hours of the day will be styled differently, so they are noticeable across the timeline as non-working time.

Reviewer: ```suggestion
The Telerik UI for .NET MAUI Scheduler control lets you define the daily business hours during the working week in the different Views. This allows you to adjust the work start and end times and the remaining hours of the day will be styled differently so users can recognize them as non-working time.
```

> The control exposes a `IsWeekendVisible` property which defines a value that indicating whether the weekend days are visible, so that only the work week is displayed.

Reviewer: ```suggestion
The control exposes a `IsWeekendVisible` property which defines a value that indicates whether the weekend days are visible so that only the work week is displayed.
```

The example below shows how to set the non-working hours in the `RadScheduler`:

>  <snippet id='' />

Reviewer: missing code and images
